src/stratigraphy/benchmark/score.py

```diff
@@ -139,15 +161,46 @@ def add_ground_truth_to_predictions(predictions: dict, ground_truth_path: Path)
         return predictions, {}
 
     number_of_truth_values = {}
+    evaluations = defaultdict(dict)
     for file, file_predictions in predictions.items():
         ground_truth_for_file = ground_truth.for_file(file)
         number_of_truth_values[file] = len(ground_truth_for_file.descriptions)
         for page in file_predictions:
-            for layer in file_predictions[page]["layers"]:
-                layer["material_description"]["is_correct"] = ground_truth_for_file.is_correct(
-                    layer["material_description"]["text"]
-                )
-    return predictions, number_of_truth_values
+            breakpoint()
+            evaluations[file][page] = ground_truth_for_file.evaluate(file_predictions[page])
+
+    return evaluations, number_of_truth_values
```

Comment:
Method not used anymore?

Reply:
The whole function is not used any more. You are right, it should be removed.